cli/flox/src/commands/activate.rs

```diff
@@ -399,6 +406,12 @@ impl Activate {
             )?;
         }
 
+        let interpreter = if self.use_old_interpreter {
```

Comment:
This conditional is actually backwards, that's why the tests mostly passed. Been trying to figure out why yours passed tests and mine didn't for a hot second lol